mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/infrastructure/persistence/store/fake/FakeDocumentReference.kt

```diff
@@ -23,6 +23,9 @@ class FakeDocumentReference(id: FakeDocumentId) : DocumentReference, CollectionB
 
   val state = MutableStateFlow(State(id))
 
+  override val id: String
```

Comment:
[Question] I was always confused when it comes to overriding in Kotlin. When do we explicit the get ?

Reply:
It depends on the behaviour you want to give. Long story shot, `override val id = "something"` creates a field (and **stores the value**) in the object, so it's a property. `override val id get() = "something"` works like a function and if what's in the `get()` changes over time, `id` may return different values if it's called multiple times.